lib/logstash/inputs/http.rb

```diff
@@ -144,6 +146,9 @@ def register
     require "logstash/inputs/http/message_handler"
     message_handler = MessageHandler.new(self, @codec, @codecs, @auth_token)
     @http_server = create_http_server(message_handler)
+
+    @remote_host_target_field ||= ecs_select[disabled: "host", v1: "[host][ip]"]
+    @request_headers_target_field ||= ecs_select[disabled: "headers", v1: "[@metadata][http][header]"]
```

Comment:
if this is still a structure containing multiple values I would vote for naming it `headers` :wink: 
in the future (when ECS ships [`http.request.headers`](https://github.com/elastic/ecs/issues/232#issuecomment-776229716)) we will likely expose the field ...

also, for `@metadata` we kind of started using a prefix convention ([example](https://github.com/logstash-plugins/logstash-input-beats/blob/v6.1.3/lib/logstash/inputs/beats.rb#L166)) when names could potentially collide,
e.g. if both http input and filter would be used in the same pipeline and they both exported  `headers` meta-data.
```suggestion
    @request_headers_target_field ||= ecs_select[disabled: "headers", v1: "[@metadata][input][http][headers]"]
```
or rather to make it more explicit, since at some point we might want to expose response headers in meta-data:
```suggestion
    @request_headers_target_field ||= ecs_select[disabled: "headers", v1: "[@metadata][input][http][request][headers]"]
```
wdyt?

Reply:
I like the second one to be more explicit in that it is a request header. 👍  thanks for the suggestion